app/models/transaction_type.rb

```diff
@@ -45,55 +50,39 @@ def self.for_income_type?(transaction_type_name)
     income_for(transaction_type_name).any?
   end
 
-  def self.other_income
-    TransactionType.where(other_income: true)
-  end
-
-  def label_name(journey: :citizens)
-    I18n.t("transaction_types.names.#{journey}.#{name}")
-  end
-
   def self.for_outgoing_type?(transaction_type_name)
     outgoing_for(transaction_type_name).any?
   end
 
-  def self.find_with_children(*ids)
-    all_ids = (ids + TransactionType.where(parent_id: ids.compact).pluck(:id)).flatten
-    where(id: all_ids)
+  def self.other_income
+    where(other_income: true)
   end
 
-  def self.any_type_of(name)
-    top_level_id = TransactionType.find_by(name:)&.id
-    return [] if top_level_id.nil?
-
-    find_with_children(top_level_id)
+  def label_name(journey: :citizens)
+    I18n.t("transaction_types.names.#{journey}.#{name}")
   end
 
   def providers_label_name
     label_name(journey: :providers)
   end
 
-  def excluded_benefit?
-    name == EXCLUDED_BENEFITS
+  def disregarded_benefit?
+    name.in?(DISREGARDED_BENEFITS)
   end
 
   def child?
     parent_id.present?
   end
 
-  def parent?
-    TransactionType.where(parent_id: id).any?
-  end
-
-  def parent
-    TransactionType.find_by(id: parent_id)
-  end
```

Comment:
These methods aren't used anywhere else, so I've removed them in the interests of keeping the public API small.